build/dockerfiles/rhel.Dockerfile

```diff
@@ -33,8 +33,8 @@ RUN yarn install
 COPY packages/ /dashboard/packages
 RUN yarn build
 
-# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/nodejs-12:1-90
-FROM registry.access.redhat.com/ubi8/nodejs-12:1-90
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/nodejs-12:1-90.1626843814
```

Comment:
@mkuznyetsov Do you happen to know if we need tag here? Or even just repo as for builder?

Reply:
https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/nodejs-12:1-90.1626843814 leads to 404, so fix https://github.com/eclipse-che/che-dashboard/pull/316